kqueen_ui/asset/dynamic/js/kqueen.js

```diff
@@ -24,31 +24,24 @@ function selectSwitch(selector) {
 };
 
 $(document).ready(function() {
-    if (location.hash) {
-        $("a[data-tabcode='" + location.hash + "']").tab("show");
-    }
-    $(document.body).on("click", "a[data-toggle]", function(event) {
-        location.hash = this.getAttribute("data-tabcode");
-    });
-
-    // Hide flash message after 3 seconds
-    setTimeout(function () {
-        $(".alert.fade").alert('close');
-    }, 5000);
+  if (location.hash) {
+    $('a[data-tabcode="' + location.hash + '"]').tab('show');
+  }
+  $(document.body).on('click', 'a[data-toggle]', function(event) {
+    location.hash = this.getAttribute('data-tabcode');
+  });
 
-    $.validator.addMethod(
-        "uuid",
-	function(value, element, condition) {
-	    var result  = /^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/i.test(value)
-	    return this.optional(element) || result == condition
-	},
-	"Entered value is not a valid UUID."
-    );
+  // Hide flash message after 7 seconds
+  setTimeout(function () {
+    $('.alert.fade').alert('close');
+  }, 7000);
 
+  $.validator.addMethod('uuid', validateFieldIsUuid, 'Entered value is not a valid UUID.');
```

Comment:
should keep all validator adding in one place and for me it's better to do in js file rather then html (can we move everything in validators.js?

Reply:
If you place them into validators.js, then they will be loaded on each page. Any reasons for this?